packages/fxa-settings/src/pages/ResetPassword/en.ftl

```diff
@@ -13,4 +13,4 @@ reset-password-warning-message-2 = <span>Note:</span> When you reset your passwo
 reset-password-button = Begin reset
 reset-password-success-alert = Password reset
 reset-password-error-general = Sorry, there was a problem resetting your password
-reset-password-error-unknown-account = Unknown account
+reset-password-error-unknown-account = Unknown account. <a>Sign up</a>
```

Comment:
This string will need a new FTL id to trigger a localization update. Also, perhaps using `<link>` as an element name would be clearer for localizers? 

FWIW, the name used here can be anything, as long as it matches the name given in the `elems` attribute